README.md

```diff
@@ -1 +1,87 @@
-# rails-zoo
\ No newline at end of file
+# Rails Zoo
+In this project, you will create a new rails application to track the animals in your Rails Zoo!
+
+You will practice the Rails basics in a new context. To keep the requirements manageable, this application will cover only the **CR** pieces of **CRUD**. This is an individual Stage 2 project.
+
+Each requirements section of this assignment has three sections: **think, create and verify**.
+- Use the **think** section to look up and test your recall on these topics. Each answer is directly relevant to the steps you need to execute in the **create** section.
+- The **create** section contains what you would see in a "normal" project's requirements.
+- The **verify** section contains the steps to verify your work in the **create** section. This gives you a moment to double check your work before moving on to the next requirement.
+
+
+
+## Setup
+1. Fork and clone this project
+2. Create a new rails application _within this folder_ using  `rails new .`
+
+## Primary Requirements
+### Create Controller & Model
+**Think**:
+- What Rails CLI command do we use to generate a new controller? Are controllers supposed to be singular or plural?
+- What Rails CLI command do we use to generate a new model? Are models supposed to be singular or plural? Where do we put the fields in the migration?
+
+**Create**:
+1. a rails controller with the name `animals`
+1. a new model (migration) with the name `animal`
+  - each animal shall have a `name`, `type` and an `age` (add any other fields you want to track about the animals)
+  - i.e. Georgia is a Lion  who is 4 years old
+1. create the schema from the migration by running `rails db:migrate`
+
+**Verify**:
+1. Check the `schema.rb` file and ensure all fields you expect are in the animals table
+  - if they are not, you may need to create another migration, or reset the database by running `rails db:reset`
+1. Run the `rails console` from the command line and create a few new animals and save them to the database. These animals will be used in the next set of requirements
+
+### View All Animals
+**Think**:
+- Which route (including controller#action) do you use for viewing _all_ of a given resource?
+- Which _model method_ do you use to retrieve _all_ of the items from the database?
```

Comment:
Would it be better to say "every item" instead of "all of the items" here?

On the one hand, a lot more students will clue into the answer for this right away if we say "all", and that's kind of the point of having a method with such an obvious name. On the other hand, if we're basically giving the answer in the question it doesn't seem like it involves much thinking.